pkg/onit/cluster/gui.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+const (
+	guiType    = "gui"
+	guiImage   = "onosproject/onos-gui:latest"
+	guiService = "onos-gui"
+	guiPort    = 80
+)
+
+func newGui(client *client) *Gui {
+	return &Gui{
+		Service: newService(guiService, []Port{{Name: "grpc", Port: guiPort}}, getLabels(guiType), guiImage, nil, nil, client),
```

Comment:
is `grpc` here correct ?

Reply:
It looks like is correct but I still need to check it
https://github.com/onosproject/onos-test/blob/1.0/pkg/onit/k8s/gui.go#L74